srcpkgs/protonmail-bridge/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'protonmail-bridge'
+pkgname=protonmail-bridge
+version=1.1.4
+revision=1
+only_for_archs="x86_64"
+build_style=fetch
+short_desc="ProtonMail Bridge for use with E-mail software"
+maintainer="Rich G <rich@richgannon.net>"
+license="Proprietary"
+homepage="https://protonmail.com/bridge"
+distfiles="https://protonmail.com/download/beta/protonmail-bridge_${version}-1_amd64.deb"
+checksum=499fbb201dc8434ad8e3b401cd68373066065fed65dc2612568307e95b63e9d5
+
+restricted=yes
+noverifyrdeps=yes
```

Comment:
nopie=yes too

Reply:
Thanks maxice8.  The package now builds.  I have installed it on my local system and need to test it out and will give feedback to the devs.  Hopefully will return with a non-proprietary license and better support from them.